package.json

```diff
@@ -36,30 +36,29 @@
     "eslint-plugin-import": "^2.17.2",
     "eslint-plugin-react": "^7.12.4",
     "fast-async": "^7",
-    "jsdom": "^15.0.0",
+    "jsdom": "^15.1.1",
     "jsdom-global": "^3.0.2",
     "mocha": "^6.1.4",
-    "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-redux": "^7.0.2",
     "react-test-renderer": "^16.8.6",
-    "redux": "^4.0.1",
+    "redux-mock-store": "^1.5.3",
     "sinon": "^7.3.2",
     "sinon-chai": "^3.3.0"
   },
   "dependencies": {
-    "@babel/runtime": "^7.4.3",
+    "@babel/runtime": "^7.4.5",
     "history": "^4.9.0",
     "little-router": "2.3.2",
     "parse-location": "^1.0.5",
-    "prop-types": "^15.7.2"
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-redux": "^7.1.0",
+    "redux": "^4.0.1"
   },
   "peerDependencies": {
-    "react": "^16.5.0",
-    "react-redux": "^4.0.0 || ^5.0.0",
-    "redux": "^3.7.2"
+    "react": "16.8.6"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=11.9.0"
```

Comment:
Should we just be updating this all the way to 12.2.0 since both ag and rent are now on that?

Reply:
ag rolled back to 11 when we were diagnosing memory issues

Reply:
ah, thats fine then.